src/qt/coincontroldialog.cpp

```diff
@@ -928,3 +928,11 @@ void CoinControlDialog::updateView()
     sortView(sortColumn, sortOrder);
     ui->treeWidget->setEnabled(true);
 }
+
+void CoinControlDialog::updateCoinControlLabels()
```

Comment:
Please change the name to `updateDialog()` or `refreshDialog()`. The method is updating/refreshing the entire component, items recreation included, and not only the labels.

If you need a hand with git let me know and can do it for you.

Reply:
Renamed refreshDialog();
Commited/squashed to 1 commit.
Anything else?